code/src/nuvla/job/actions/application_docker_compose_validate.py

```diff
@@ -19,15 +19,30 @@ def __init__(self, _, job):
 
     @staticmethod
     def get_env_to_mute_undefined(module_content):
-        value = '1'  # Some value that make docker-compose config happy, even for port number field
-        env_variables = {'NUVLA_DEPLOYMENT_ID': value,
-                         'NUVLA_DEPLOYMENT_UUID': value,
-                         'NUVLA_API_KEY': value,
-                         'NUVLA_API_SECRET': value,
-                         'NUVLA_ENDPOINT': value}
+        aux_port_value = 1  # Some value that make docker-compose config happy, even for port number field
+        aux_volume_value = '/tmp/{}/'
```

Comment:
This line is adding warnings in the code reported by sonarcloud: https://sonarcloud.io/project/security_hotspots?id=nuvla_job-engine&pullRequest=232&resolved=false&types=SECURITY_HOTSPOT
Since it's just a fake value, could it be something else that doesn't make sonarcloud complain.